### DISCUSSION TITLE
`witdh` could be either `with` or `width`

### DISCUSSION THREAD

**Discussion by hello-smile6** [SpellCheckJS/SpellCheckJS#3] [Entry Review]
https://github.com/SpellCheckJS/SpellCheckJS/blob/c3cd91fecc09cba04574f91b9ce641fd8556d69f/spellcheck.json#L2
Should this be removed?

**Comment by hello-smile6** [comment]
@ScolderCreations What do you think?

**Reply by ScolderCreations** [reply]
It seems more likely to be width. You should implement some sort of list system, where it can pick from a list of matches.

**Reply by hello-smile6** [reply]
> It seems more likely to be width. You should implement some sort of list system, where it can pick from a list of matches.

It's supposed to be 1-1. I'd have to rewrite the HTML client, and it'd make it significantly harder for the development of future clients. Also, you can't exactly show the end-user a prompt when rewriting a webpage JIT in production and ask them which they'd want.

**Comment by hello-smile6** [comment]
@webdev03 What'd you say?
